WordPress/Sniffs/VIP/PostsPerPageSniff.php

```diff
@@ -24,6 +24,15 @@
 class PostsPerPageSniff extends AbstractArrayAssignmentRestrictionsSniff {
 
 	/**
+	 * Posts per page property
+	 *
+	 * Posts per page limit to check against.
+	 *
```

Comment:
Missing `@since` tag.
It would also be a good idea to add a note about this change to the sniff class changelog at the top of the file.

Reply:
The correct since tag is `0.3.0` or `0.13.0`? There are two since tags above the class definition, not sure which one to use.

Reply:
The correct since tag would be `0.14.0` as that's the release in which this property is added ;-)

Reply:
Sure thing :D Fixed